src/parser/mage/frost/modules/checklist/Component.js

```diff
@@ -46,18 +47,38 @@ class FrostMageChecklist extends React.PureComponent {
             </>
           )}
         >
           <Requirement name="Used Brain Freeze procs" thresholds={thresholds.brainFreezeUtilization} tooltip="Your Brain Freeze utilization. Brain Freeze is your most important proc and it is very important that you utilize them properly." />
           <Requirement name="Used Fingers of Frost procs" thresholds={thresholds.fingersOfFrostUtilization} />
-          <Requirement name="Ice Lance into Winter's Chill" thresholds={thresholds.wintersChillIceLance} tooltip="Using Brain Freeze will apply the Winter's Chill debuff to the target which causes your spells to act as if the target is frozen. Therefore, you should always cast Ice Lance after every instant cast Flurry so that the Ice Lance hits the target while Winter's Chill is up." />
-          <Requirement name="Hardcast into Winter's Chill" thresholds={thresholds.wintersChillHardCasts} tooltip="Flurry travels faster than your other spells, so you can pre-cast Frostbolt, Ebonbolt, or Glacial Spike before using your instant cast Flurry. This will result in the pre-cast spell landing in the Winter's Chill debuff and dealing bonus shatter damage." />
+          
+        </Rule>
+        <Rule
+          name="Use Glacial Spike properly"
+          description={(
+            <>
+              <SpellLink id={SPELLS.GLACIAL_SPIKE_TALENT.id} /> is one of the most impactful talents that you can choose and it plays a large part in your rotation; So you should always ensure that you are getting the most out of it, because a large part of your damage will come from making sure that you are handling Glacial Spike properly. As a rule, once you have Glacial Spike available, you should not cast it unless you have a <SpellLink id={SPELLS.BRAIN_FREEZE.id} /> proc to use alongside it (<SpellLink id={SPELLS.GLACIAL_SPIKE_TALENT.id} /> > <SpellLink id={SPELLS.FLURRY.id} /> > <SpellLink id={SPELLS.ICE_LANCE.id} />) or if you also have the <SpellLink id={SPELLS.SPLITTING_ICE_TALENT.id} /> and the Glacial Spike will hit a second target. If neither of those are true, then you should continue casting <SpellLink id={SPELLS.FROSTBOLT.id} /> until you have a <SpellLink id={SPELLS.BRAIN_FREEZE.id} /> proc. If you are consistently in situations where you are waiting to get a Brain Freeze proc, then consider taking the <SpellLink id={SPELLS.EBONBOLT_TALENT.id} /> talent and saving it for when you need to generate a proc to use with Glacial Spike.
+            </>
+          )}
+        >
+          {combatant.hasTalent(SPELLS.GLACIAL_SPIKE_TALENT.id) && <Requirement name="Glacial Spike utilization" thresholds={thresholds.glacialSpikeUtilization} />}
```

Comment:
You added glacial spike utilization here -- but it's also still in the Talents section, is that intentional?